malariagen_data/anoph/base.py

```diff
@@ -397,7 +397,16 @@ def _read_sample_sets(self, *, single_release: str):
     @check_types
     @doc(
         summary="Access a dataframe of sample sets",
-        returns="A dataframe of sample sets, one row per sample set.",
+        returns="""A dataframe of sample sets, one row per sample set. It contains five columns:
+         **sample_set** is the name of the sample set,
+         **sample_count** is the number of samples the sample set contains,
+         **study_id** is the identifier for the study that generated the sample set,
+         **study_url** is the URL of the study on the MalariaGEN website,
+         **term_of_use_expiry** is the date when the terms of use expire,
+         **terms_of_use_url** is the URL of the terms of use,
+         **release** is the identifier of the release containing the sample set,
+         **unrestricted_use** whether the sample set can be without restriction (e.g., if the terms of use have expired).
```

Comment:
Swap \*\* for \`.

Reply:
Yep, it took me 3 files to figure out how to do a find and replace on my Mac ;). I should have rechecked the files where I used another method.